dependencies.gradle

```diff
@@ -50,20 +52,23 @@ project.ext.packages = [
       slf4j: "org.apache.logging.log4j:log4j-slf4j-impl:${versions.log4j}",
     ],
     opencsv: "net.sf.opencsv:opencsv:${versions.opencsv}",
+    google_truth: "com.google.truth:truth:${versions.google_truth}",
```

Comment:
Nice!

Reply:
You can thank the testing for Checkstyle for that, as I noted in a comment, "Google Truth is needed for checkstyle tests (as of checkstyle 9.2.1)".